src/alchemy-sdk.ts

```
@@ -389,8 +389,7 @@ async function postNftRest<T>(
 function toAlchemyError(e: unknown, context: string): Error {
   if (e instanceof AxiosError) {
     const data = e.response?.data as
-      | { message?: string; error?: string | { message?: string } }
-      | undefined;
+      { message?: string; error?: string | { message?: string } } | undefined;
```

ESLint

[error] Replace `{·message?:·string;·error?:·string·|·{·message?:·string·}·}` with `|·{·message?:·string;·error?:·string·|·{·message?:·string·}·}⏎·····`

(prettier/prettier)